Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM alpine
```

Comment:
FYI you're missing copyright headers

Also alpine version needs to be specified like `FROM alpine:<tag>`

Reply:
ok